context.js

```diff
@@ -15,16 +15,15 @@ var megaman = new Robot("Mega Man");
 var bmo = new Robot("B-Mo");
 var wall_e = new Robot("Wall-E");
 
-
 // #1
-console.log(  );
-
-// #2  
-console.log(  );
+console.log(add(null, 0, 1));
```

Comment:
So this is just a regular function call. We don't need to pass in any context here.